python/nammu/view/AtfAreaView.py

```diff
@@ -313,13 +318,14 @@ def undoableEditHappened(self, event):
 
         # If significant INSERT/REMOVE event happen, end and add current
         # edit compound to undo_manager and start a new one.
-        if (edit_type == "INSERT" or edit_type == "REMOVE") and not self.must_compound:
-            # Explicitly end compound edits so their inProgress flag goes to
-            # false. Note undo() only undoes compound edits when they are not
-            # in progress.
-            self.current_compound.end()
-            self.current_compound = CompoundEdit()
-            self.undo_manager.addEdit(self.current_compound)
+        if ((edit_type == "INSERT" or edit_type == "REMOVE") and
+                not self.must_compound):
+                # Explicitly end compound edits so their inProgress flag goes
```

Comment:
Do you mean the body inside the if? The second part of the condition after the `and` clause would make PEP8 complain if I had 4-spaces indent due to 

```
E128 continuation line under-indented for visual indent
```

or 

```
E129 visually indented line with same indent as next logical line
```

depending on how many spaces I indented with.